<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>800</height>
    </size>
   </property>
   <widget class="QFrame" name="top_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>150</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_profilePicture">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>180</width>
        <height>180</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>180</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../../.designer/Practica 7/Icons/Icon_Fish.jpg</normaloff>../../../../../../.designer/Practica 7/Icons/Icon_Fish.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>165</width>
        <height>165</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>5</y>
       <width>390</width>
       <height>190</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>390</width>
       <height>190</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>390</width>
       <height>190</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="userName">
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>370</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>NOMBRE USUARIO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="menu_Status">
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>370</width>
          <height>25</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Disponible</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ocupado</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ausente</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Desconectado</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolBox" name="menu_Account">
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>370</width>
          <height>100</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="options">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>353</width>
           <height>102</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Cuenta</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="change_Username">
            <property name="text">
             <string>Cambiar Apodo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="change_profilePicture">
            <property name="text">
             <string>Cambiar Foto de Perfil</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_Sesion">
            <property name="text">
             <string>Cerrar Sesi√≥n</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="buttons_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>150</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_Tereneitor">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>5</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="text">
      <string>Terreneitor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_smartHome">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>5</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="text">
      <string>Casa Inteligente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Matrix">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="text">
      <string>Matriz de Leds</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="list_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>600</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolBox" name="contactList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>105</y>
       <width>580</width>
       <height>90</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>580</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>580</width>
       <height>90</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>60</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Contactos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="btn_Private">
         <property name="text">
          <string>AbrirPrivado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QToolBox" name="groupList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>580</width>
       <height>90</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>580</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>580</width>
       <height>90</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>60</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Grupos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_Group">
         <property name="text">
          <string>AbrirGrupo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>600</width>
      <height>250</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>250</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>250</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="msgView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>580</width>
       <height>210</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>580</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>580</width>
       <height>210</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="msgWrite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>500</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="msgSend">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>219</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionsalir">
   <property name="text">
    <string>Cerrar Sesi√≥n</string>
   </property>
  </action>
  <action name="actioncambiar_nombre_de_usuario">
   <property name="text">
    <string>cambiar nombre de usuario</string>
   </property>
  </action>
  <action name="action_mas_opciones">
   <property name="text">
    <string>&amp;mas opciones</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
